src/src/program-coordinator/program-coordinator.service.ts

```diff
@@ -2244,4 +2246,91 @@ export class ProgramCoordinatorService {
 			});
 		}
 	}
+
+	async programCoordinatorLearnerVerification(body, request, response) {
+		let query;
+		let hasura_response;
+		let validation_result;
+		let pc_user_id = request?.mw_userid;
+		let { user_id, enrollment_verification_status } = body;
+		let set_update_body;
+
+		if (!user_id) {
+			return response.json({
+				status: 422,
+				message: 'Provide valid user id ',
+				data: {},
+			});
+		}
+
+		query = `query MyQuery {
+			program_beneficiaries(where: {user_id: {_eq: ${user_id}},facilitator_user: {program_faciltators: {pc_id: {_eq: ${pc_user_id}}}}}) {
+			  user_id
+			  id
+			}
+		  }
+		  `;
```

Comment:
need to check cohor and state (program_id)

Reply:
resolved